mozetl/search/aggregates.py

```diff
@@ -171,17 +171,29 @@ def get_search_addon_version(active_addons):
     )
 
 
+def get_ad_click_count(ad_click_count):
+  if ad_click_count:
+    return [{'engine': e, 'source': "ad-click:", 'count': c} for e, c in ad_click_count.items()]
+  return []
```

Comment:
To confirm, users without ad-clicks will show up as `NULL`, correct? We impute search counts for 0 search users, but I don't think we can do that for ad-clicks since a large portion of clients don't have ad-click telemetry yet.

Reply:
Yup.

Reply:
Correct, I think.